packages/blade/.storybook/react/manager-head.html

```diff
@@ -1,3 +1,59 @@
 <link rel="preconnect" href="https://fonts.googleapis.com">
 <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
 <link href="https://fonts.googleapis.com/css2?family=Lato:wght@400;700&display=swap" rel="stylesheet">
+<link rel="stylesheet"
+  href="https://cdn.jsdelivr.net/gh/katorlys/prism-theme-github/themes/prism-theme-github-light.css">
+<link rel="icon"
+  href="https://raw.githubusercontent.com/razorpay/blade/1e77f0b35172654037a431a916b3190b545fd232/branding/favicon.ico" />
+<style>
+  :root {
+    --blade-surface-action-icon: rgba(163, 175, 191, 1);
+    --blade-surface-action-icon-active: rgba(93, 109, 134, 1);
+    --blade-surface-text-normal: rgba(19, 38, 68, 1);
+    --blade-brand-gray: rgba(121, 135, 156, 0.18);
+    --blade-brand-gray-light: rgba(121, 135, 156, 0.08);
+  }
+
+  .search-field input {
+    border-radius: 4px !important;
+  }
+
+  .sidebar-subheading,
+  .sidebar-subheading svg {
+    color: rgba(19, 38, 68, 1) !important;
+    font-weight: bold !important;
+    text-transform: none !important;
+    letter-spacing: normal !important;
+    font-size: 12px !important;
+  }
+
+  .sidebar-item:hover {
+    background: var(--blade-brand-gray-light) !important;
+  }
+
+  .sidebar-item svg {
+    color: var(--blade-surface-action-icon) !important;
+  }
+
+  .sidebar-item[data-selected="true"] {
+    background: var(--blade-brand-gray) !important;
+    color: var(--blade-surface-text-normal) !important;
+    font-weight: bold !important;
+  }
+
+  .sidebar-item[data-selected="true"]+a[href] {
+    opacity: 0 !important;
+  }
+
+  .sidebar-item[data-selected="true"]+a[href]:focus {
+    opacity: 1 !important;
+  }
+
+  .sidebar-item[data-selected="true"] svg {
+    color: var(--blade-surface-action-icon-active) !important;
+  }
+
+  .sto-1va4tas {
```

Comment:
I hope this isn't a dynamic style

Reply:
no I too checked. It stays static. This is how CRED has also overriden styles of their storybook.

I have anyway tried to avoid this until I couldn't so wouldn't break too many things even if it changes.